README.md

```diff
@@ -1,2 +1,66 @@
-# licensespring-obfuscation
-Showcasing how this works
+# licensespring-obfuscation-sample
+
+This is a sample project demonstrating the use of LicenseSpring in a Java application. The project utilizes the LicenseSpring License Client library to manage license activation, checking, and deactivation.
+
+## Prerequisites
+
+- Java Development Kit (JDK) 8 or higher
+- Apache Maven
+
+## pom.xml
+
+The project is built using Maven, and the dependencies are specified in the `pom.xml` file. The main dependencies include:
+
+- `licensespring-license-client`: LicenseSpring License Client library
+- `junit`: A testing framework for Java applications
+
+The `pom.xml` also includes the `maven-assembly-plugin` to generate a JAR file with all dependencies and the `proguard-maven-plugin` to obfuscate the final JAR file.
+
+
+## MyLicenseManager
+
+The `MyLicenseManager` class is a simple wrapper around the LicenseSpring License Client library. It demonstrates license activation, checking, and deactivation.
+
+
+## App (Main Class)
+
+The `App` class is the main entry point of the application. It expects five command-line arguments: `apiKey`, `sharedKey`, `licenceKey`, `productName`, and `isStaging` (yes/no).
```

Comment:
you can deafult "isStaging" param to "no" in case it's not part of the input arguments -> this will make it an optional param (which it should be)